### PR TITLE
Add search params read for send many

### LINKED ISSUE #31: URL params recipient, amount, token
I was thinking of having a url format for sending amounts to a list of recipients with amounts. 
The benefit of that being we could use sendstx as a general service for transferring tokens for the whole community.
For consumers they could use a helper function to generate a compliant URL an example implementation that can construct the url for send many including the token name and a list of recipients like
```ts
const params = new URLSearchParams([
  ["recipient", 'SP1...MZ,1000,"thanks"'],
  ['recipient', 'hz.btc,1000,"shokran"'],
  ['recipient', 'friedger.btc,1000,"danke"'],
])

const sendSTXURL = `https://sendstx.com/not?${params.toString()}`
```
You might ask why is that and my purpose from this is to have a trusted place where people can send tokens that might help me in my personal project delegate trust of deposits to sendstx I want to support contract addresses in the future as well.

We could even go further with this by providing support for arbitrary tokens like `sendstx.com/sp...kkz.nope?params` and the app can fetch the abi of the contract to check that there is a compatible send-many function and if so use that and secure it with post conditions, decimals as well can be retrieved by using the `get-decimals` function.

### PR DESCRIPTION
**Pull request by hozzjss**
This PR should add the feature to fetch the data from the url 
Closes #31

### CODE REVIEW

**Review by friedger — COMMENTED** [review]
Looks good! Just one question before merging.

File: src/components/SendManyInputContainer.tsx [hunk 1]
```diff
@@ -114,6 +114,27 @@ export function SendManyInputContainer({
   const [namesResolved, setNamesResolved] = useState(true);
   const [firstMemoForAll, setFirstMemoForAll] = useState(false);
 
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (hozzjss) [reply src/components/SendManyInputContainer.tsx:117]: This should run only once when page loads get all the parameters and fill up the rows, there is an improvement that can be made here if the project switches to `react-router-dom` where we'd watch the changes in search and have a bi-directional data flow whereby the changes in search updates the component state and the other way around
However, with the current reach router that im not sure is possible

File: src/components/SendManyInputContainer.tsx [hunk 2]
```diff
@@ -114,6 +114,27 @@ export function SendManyInputContainer({
...
+    const search = new URL(window.location.href).searchParams;
+    const rows = search.getAll('recipient').map(item => {
+      const { amount, recipient, memo } =
```
[the remark this answers is not in the document]
Comment (hozzjss) [reply src/components/SendManyInputContainer.tsx:120]: Great Idea I'll push something for you to review

File: src/components/SendManyInputContainer.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (friedger) [reply src/components/SendManyInputContainer.tsx:117]: The code will run once without using useEffect

File: src/router.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { createRoot } from 'react-dom/client';
+import { createBrowserRouter, RouterProvider, Route, Link } from 'react-router-dom';
+
+const router = createBrowserRouter([
```
Comment (friedger) [inline src/router.tsx:5]: This can be removed or the content of Apps.tsx moved to here.

**Review by friedger — CHANGES_REQUESTED** [review]
Looks good. Just the router.tsx file needs a fix.

**Review by friedger — APPROVED** [review]

**Pull request merged**